From c8c97ecce1a14130fbd5f8c5c724e5fd612c28b8 Mon Sep 17 00:00:00 2001
From: huzhen <george.hu@spacemit.com>
Date: Thu, 7 Nov 2024 10:58:47 +0800
Subject: [PATCH 1248/1448] dts: modify the address space allocation of
 pcie2_rc

  Increase prefetchable space allocation to support
  pcie graphics devices.

Change-Id: Ifbc154a0bdf1d10407b38d378470bd0306746726
---
 arch/riscv/boot/dts/spacemit/k1-x.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x.dtsi b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
index 6b80684a38a4..7d6c83a344d1 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x.dtsi
+++ b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
@@ -2297,7 +2297,8 @@ pcie2_rc: pcie@ca800000 {
 			#address-cells = <3>;
 			#size-cells = <2>;
 			ranges = <0x01000000 0x0 0xb7002000 0 0xb7002000 0x0 0x100000>,
-				 <0x02000000 0x0 0xa0000000 0 0xa0000000 0x0 0x17000000>;
+				 <0x42000000 0x0 0xa0000000 0 0xa0000000 0x0 0x10000000>,
+				 <0x02000000 0x0 0xb0000000 0 0xb0000000 0x0 0x7000000>;
 			interconnects = <&dram_range2>;
 			interconnect-names = "dma-mem";
 
-- 
2.47.0

